src/pages/Account/Tabs/TransactionsTab.tsx

```diff
@@ -1,33 +1,134 @@
-import React from "react";
-import {Alert} from "@mui/material";
+import React, {useState} from "react";
+import {Pagination, Box} from "@mui/material";
 import TransactionsTable from "../../Transactions/TransactionsTable";
 import Error from "../Error";
 import {useGetAccountTransactions} from "../../../api/hooks/useGetAccountTransactions";
+import EmptyTabContent from "../../../components/IndividualPageContent/EmptyTabContent";
+import {useGlobalState} from "../../../GlobalState";
+import {ResponseError} from "../../../api/client";
+import {useQuery} from "react-query";
+import {Types} from "aptos";
+import {getAccount} from "../../../api";
 
-type TransactionsTabProps = {
+const TXN_PER_PAGE = 50;
+
+function getPageStartSequenceNumbers(sequenceNum: number): number[] {
+  const pageStarts: number[] = [];
+
+  const numOfPages = Math.ceil(sequenceNum / TXN_PER_PAGE);
+  let num = sequenceNum;
+  for (let i = 0; i < numOfPages; i++) {
+    num = num - TXN_PER_PAGE;
+    // TODO:
+    // There's a bug in the API when `start` is 0,
+    // getAccountTransactions will return the latest transactions.
+    // Instead, it should return the oldest transactions (starts 0).
+    // This is a walk around to show the oldest transactions in the last page. Starts from 1.
+    // This way we will lose the transaction of sequence number 0,
+    // but at least we'll have everything else in the last page.
+    // Once the bug is fixed on the API side,
+    // we should change the next line to: num = num >= 0 ? num : 0;
+    num = num >= 1 ? num : 1;
+    pageStarts.push(num);
+  }
+
+  return pageStarts;
+}
+
+type TransactionsPaginationTableProps = {
   address: string;
+  sequenceNum: number;
 };
 
-export default function TransactionsTab({address}: TransactionsTabProps) {
-  const {isLoading, data, error} = useGetAccountTransactions(address);
+function TransactionsPaginationTable({
+  address,
+  sequenceNum,
+}: TransactionsPaginationTableProps) {
+  const [currentPageNum, setCurrentPageNum] = useState<number>(1);
```

Comment:
I recommend reusing the pagination logic from the other transactions table, including having `start` in the query string (instead of storing the page number as a state variable).

Reply:
I thought about make `start` as a url param. However, this table is under a tab in the account page. Ideally we should have something like `/account/0x12345/transactions?start=123`. But since don't have the tab part `/transactions` yet, it'd be kinda weird to have `/account/0x12345?start=123` because other tabs in this page won't share the same `start`. 

Let me add a TODO here to add url param for both tabs and pagination. This PR is needed asap because without it users won't be able to see all their transactions. Thanks!